coverage.py

```diff
@@ -395,26 +396,47 @@ class CoverageProvider(BaseCoverageProvider):
     Turns errors in processing into coverage records with failure flags.
     """
 
-    # Does this CoverageProvider get its data from a source that also
-    # provides licenses for books?
-    CAN_CREATE_LICENSE_POOLS = False
```

Comment:
This is no longer relevant. A CoverageProvider that is instantiated with a Collection can create LicensePools; a CoverageProvider that has no Collection cannot.